### PR TITLE
dvp

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: k210.svd [hunk 1]
```diff
@@ -2434,9 +2434,21 @@
                                 <bitRange>[0:0]</bitRange>
                             </field>
                             <field>
-                                <name>user_mode</name>
-                                <description>USER_MODE</description>
+                                <name>mode</name>
+                                <description>MODE</description>
```
Comment (Disasm) [inline k210.svd]: Could you commit these `TIMER` fixes in separate PR?

File: k210.svd [hunk 2]
```diff
@@ -2567,9 +2579,269 @@
...
+                            <description>AUTO_ENABLE</description>
+                            <bitRange>[4:4]</bitRange>
+                        </field>
+                        <field>
+                            <name>burst_size_4beats</name>
+                            <description>BURST_SIZE_4BEATS</description>
+                            <bitRange>[8:8]</bitRange>
+                        </field>
+                        <field>
+                            <name>FORMAT</name>
```
Comment (Disasm) [inline k210.svd]: Maybe lowercase is more suitable here

**Review by Disasm — CHANGES_REQUESTED** [review]

**Comment by Disasm** [comment]
Sorry for me being slow and thanks for your effort! I'll be happy to merge this as soon as you fix these minor issues.

**Comment by burrbull** [comment]
Done.

**Pull request merged**